Source/ThirdParty/WPE-platform/src/westeros/WesterosViewbackendInput.cpp

```diff
@@ -296,6 +318,7 @@ void WesterosViewbackendInput::pointerHandleAxis( void *userData, uint32_t time,
         struct wpe_input_axis_event event{ wpe_input_axis_event_type_motion, e->time, coords.first, coords.second, e->axis, -wl_fixed_to_int(e->value) };
         wpe_view_backend_dispatch_axis_event(backend_input.m_viewbackend, &event);
 
+        g_ptr_array_remove_fast(backend_input.m_axisEventDataArray, data);
```

Comment:
As I've understood from the documentation (https://developer.gnome.org/glib/stable/glib-Pointer-Arrays.html#g-ptr-array-remove-fast) both of the functions g_ptr_array_remove() and g_ptr_array_remove_fast() work almost similarly, but the latter can change the order of items in the array.

In terms of memory usage, both functions behave identically.

For example:

Original array:
3, 5, 8, 13, 21, <reserved>

After g_ptr_array_remove(8):
3, 5, 13, 21, <reserved>, <reserved>

After g_ptr_array_remove_fast(8):
3, 5, 21, 13, <reserved>, <reserved>

In case of large array size the second function works faster.
In case of array with only 1 item (the most popular case) there are no difference between these functions.

We don't have to care about the order of items in the array, because we don't use array indices.

That's why I've used the function g_ptr_array_remove_fast()